### PR TITLE
dhclient: add page

### PR DESCRIPTION
**Pull request by BeLi4L**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/common/dhclient.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# dhclient
+
+> DHCP client.
+
+- Get an IP address for the eth0 interface:
```
Comment (agnivade) [inline pages/common/dhclient.md]: Put eth0 within backticks(``)

File: pages/common/dhclient.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+`sudo dhclient eth0`
```
Comment (agnivade) [inline pages/common/dhclient.md]: eth0 within tokens (`{{}}`)

File: pages/common/dhclient.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+- Release an IP address for the eth0 interface:
```
Comment (agnivade) [inline pages/common/dhclient.md]: same as above

File: pages/common/dhclient.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+`sudo dhclient -r eth0`
```
Comment (agnivade) [inline pages/common/dhclient.md]: same as above

File: pages/common/dhclient.md (hunk 1 above)
Comment (BeLi4L) [reply pages/common/dhclient.md]: Done in 7d0f7d37b298915bf4cafe2f7fc4d4732556f20d

File: pages/common/dhclient.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (BeLi4L) [reply pages/common/dhclient.md]: Done in 7d0f7d37b298915bf4cafe2f7fc4d4732556f20d

**Pull request merged**